docs/umi/tertiary-analysis.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+parent: Single cell
+title: Tertiary Analysis
+nav_order: 8
+---
+
+# Tertiary analysis of MAS-Seq data with popular community tools
+
+This is a simple tutorial on how to analyze PacBio [MAS-Seq](https://www.pacb.com/products-and-services/applications/rna-sequencing/single-cell-rna-sequencing/) data using SMRT Link v11.1. After sequencing, SMRT Link 11.1 has a push button solution to deconcatenate HiFi reads into individual full-length S-reads (transcripts), classify the novelty of isoforms against Gencode and produce a matrix that is compatible with many popular community tools such as Seurat or Scanpy.
+
+SMRT Link 11.1 is available for free on [PacBio's website](https://www.pacb.com/support/software-downloads/). If you prefer to analyze the data with command line, the workflow is described here on [https://skera.how/](https://skera.how/) for read deconcatenation and [https://isoseq.how](https://isoseq.how/umi/cli-workflow.html) for IsoSeq analysis. We recommend using SMRT Link GUI if possible as it produces many useful plots for sanity check purpose. Nonetheless, the final results from command line analysis are the same as those from SMRT Link.
```

Comment:
change "available for free" to just "available"

Reply:
"read deconcatenation" --> "read segmentation"
"for IsoSeq analysis" --> "for single-cell Iso-Seq analysis"